src/color-swatch/color-swatch.js

```diff
@@ -109,6 +111,26 @@ const Self = class ColorSwatch extends NudeElement {
 			this.style.setProperty("--color", colorString);
 		}
 
+		if (name === "coords") {
+			if (this.color) {
+				this._el.coords?.replaceChildren(); // remove all children
```

Comment:
Nit: You don't need `replaceChildren()` to remove all children, `el.textContent = ""` works and is more common.
Why the optional chaining? Is there any chance this runs before `this._el.coords` is defined? I thought we fixed that?

Reply:
> Nit: You don't need `replaceChildren()` to remove all children, `el.textContent = ""` works and is more common.

This was my first guess. But I wasn't sure it was a common practice. Now I know. Thanks!

> Why the optional chaining? Is there any chance this runs before `this._el.coords` is defined? I thought we fixed that?

It's. My bad. Will remove it.